prefect_gcp/cloud_run.py

```diff
@@ -398,6 +407,30 @@ async def run(self, task_status: Optional[TaskStatus] = None):
             )
             return result
 
+    @sync_compatible
+    async def kill(self, identifier: str, grace_seconds: int = 30) -> None:
```

Comment:
```suggestion
    async def kill(self, identifier: str, grace_seconds: Optional[int] = 30) -> None:
```

Reply:
I don't think this should be typed `Optional` since the default is not None, but let me know if I'm missing something.

Reply:
I think this changes how it displays in the UI though, right? I can't quite remember, but will be UI insist that they enter a value if it's not listed as optional?

Reply:
*My general thought is that if you gave me that value and insisted on a value, I'm going to put some number, and then I'm going to get a warning that it's not the number that it wanted. But I don't have a strong feeling here.

Reply:
This shouldn't show up in the UI since it's a method of block I think.

Reply:
🤦 oh man... My brain was pretty scattered this afternoon. Don't mind me.

Reply:
No worries, thanks for reviewing!